DOCUMENTATION.md

```diff
@@ -1,34 +1,34 @@
 # Java client documentation
 
-This documentation is for developers interested in using the GOV.UK Notify Java client to send emails, text messages or letters.
+This documentation is for developers interested in using the GOV.AU Notify Java client to send emails or text messages. This is a hard fork
+of the [GOV UK's notify java client](https://github.com/alphagov/notifications-java-client) which was used as the base for the library.
 
 # Set up the client
 
 ## Install the client
 
-The `notifications-java-client` deploys to Bintray.
+The `notify-client-java` deploys to Bintray's JCenter. You will need to include JCenter as your repository within your maven/gradle configuration. 
 
-Go to the [GOV.UK Notify Java client page on Bintray](https://bintray.com/gov-uk-notify/maven/notifications-java-client) [external link]:
+The library is available on the [GOV.AU Notify Java client page on Bintray](https://bintray.com/notify-infra/notify-jars/notify-client-java) [external link]:
 
-1. Select __Set me up!__ and use the appropriate download instructions.
-1. Go to the Maven build settings section of the page and copy the appropriate dependency code snippet.
+You can also refer to the [sample Java example](https://github.com/govau/notify-examples-java) for an example application using the library.
 
-Refer to the [client changelog](https://github.com/alphagov/notifications-java-client/blob/master/CHANGELOG.md) for the version number and the latest updates.
+Refer to the [client changelog](https://github.com/govau/notify-client-java/blob/master/CHANGELOG.md) for the version number and the latest updates.
 
 ## Create a new instance of the client
 
 Add this code to your application:
 
 ```java
-import uk.gov.service.notify.NotificationClient;
+import au.gov.notify.NotificationClient;
```

Comment:
`au.gov.notify.NotifyClient` or just `au.gov.notify.Client`

Reply:
Changed to `au.gov.notify.NotifyClient`